api/v1alpha1/dataprotectionapplication_types.go

```diff
@@ -45,6 +45,7 @@ const DefaultPluginCSI DefaultPlugin = "csi"
 const DefaultPluginVSM DefaultPlugin = "vsm"
 const DefaultPluginOpenShift DefaultPlugin = "openshift"
 const DefaultPluginKubeVirt DefaultPlugin = "kubevirt"
+const DefaultPluginHypershift DefaultPlugin = "hypershift"
```

Comment:
I think this is still not testable

CRD validation does not include `hypershift` as an option (line 37 in this file. After changing it, you need to run `make bundle` to update YAML files)